### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by tPiffrader**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,116 @@
...
+#Usage
+
+Example:
+```javascript
+var map = require("gotmap");
+map.config(object);
+// object incluse two fields:
+// apiLocation = link to the api
+// apiToken = token for post/put/delete request to api
+map.init();
```
Comment (sacdallago) [inline README.md:14]: What is the output? What will this actually produce? A div?

File: README.md [hunk 2]
```diff
@@ -1,39 +1,116 @@
...
-Install node modules:
+    - [] Editor for map
+
+# HOWTO
+##run web page:
+Install the gotmap and open the map:
+
+```shell
+npm install gotmap
+open node_modules/gotmap/mockup/mockup.html
```
Comment (sacdallago) [inline README.md:34]: `open` is specific to OsX ;) it has no meaning in any other OS

File: README.md [hunk 3]
```diff
@@ -1,39 +1,116 @@
...
-#Usage
+#
+Konstruktor([htmlEl] mapEl, [htmlEl] filterEl, [htmlEl] timelineEl, [htmlEl] sidebarEl)
+# Map API
 
-Example:
+## Modal
+Show the modal:
+```javascript
+showModal([string] id, [string] title, [string] cssclass)
```
Comment (sacdallago) [inline README.md:63]: Still missing context here. `id` of the modal? `title` will we the title of the modal I suppose.. `cssclass` I have no idea what this is.
Is this gonna generate html code or activate the modal referenced by the `id`?

File: README.md [hunk 4]
```diff
@@ -1,39 +1,116 @@
...
-map.init();
-// create a map
+Hide the modal:
+```javascript
+hideModal()
+```
+## Characters in the map
+Add a character:
+```javascript
+addCharacter([object] character)
```
Comment (sacdallago) [inline README.md:72]: Where do I get the `character` object? What does it look like?

File: README.md (hunk 1 above)
Comment (tPiffrader) [inline README.md:14]: @AlexMoroz you gave me these lines, can you fix these please? :)

File: README.md (hunk 2 above)
Comment (tPiffrader) [inline README.md:34]: @AlexMoroz here the same ;)

File: README.md (hunk 4 above)
Comment (tPiffrader) [inline README.md:72]: Refactoring is still in process. We will add it when it's done :)

**Comment by gyachdav** [comment]
These docs should go into the wiki section of your repo

**Comment by tPiffrader** [comment]
We will add it when refactoring is done :+1:

File: README.md (hunk 1 above)
Comment (AlexMoroz) [inline README.md:14]: What do you want to fix?

File: README.md (hunk 2 above)
Comment (AlexMoroz) [inline README.md:34]: open also means open =)) just open it in browser

File: README.md (hunk 1 above)
Comment (AlexMoroz) [inline README.md:14]: init run the js

**Pull request merged**